tutorials/configure-chef-ubuntu-xenial/index.mdx

```diff
@@ -103,12 +103,12 @@ You can easily manage up to 10,000 nodes using Chef. Chef also makes it easy to
     ```
     apt-get upgrade
     ```
-3. Go to the [Chef Infra Server Download Page](https://downloads.chef.io/tools/infra-server), from your local machine's browser, and click **Download** next to the version of Ubuntu you are running.
+3. Go to the [Chef Infra Server Download Page](https://community.chef.io/downloads/tools/infra-server?os=ubuntu), from your local machine's browser, and click **Download** next to the version of Ubuntu you are running.
 4. Complete your information in the form that displays and click **Download**.
-5. Right-click **Download Chef Infra Server directly** and copy the link. 
+5. Right-click **Download Chef Infra Server directly** and copy the link.
 6. Launch the installation using `wget` + the link you copied, from the terminal of your remote machine. Make sure to use your own link and not the one shown below as an example:
     ```
-    wget https://packages.chef.io/files/stable/chef-server/14.16.19/ubuntu/20.04/chef-server-core_14.16.19-1_amd64.deb
+    wget https://packages.chef.io/files/stable/chef-server/12.19.31/ubuntu/18.04/chef-server-core_12.19.31-1_amd64.deb
```

Comment:
I am not sure, the chef documentation is not very clear. The later versions seem to be paid ones and not free to use. 
This is the latest version listed on the community download page: https://community.chef.io/downloads/tools/infra-server?os=ubuntu